### DISCUSSION TITLE
Images / sounds not loading from the /public folder

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#9629] [CLI - Capacitor / Cordova modes] [answered]
While developing the app, everything was fine with loading images / sounds from the public folder but once I build it for production, the app cannot access it, e.g. doesn't display the images & doesn't play the sounds.

Maybe some issue how I define the paths?

Example: q-btn class="q-ma-none" size="1.5em" round flat icon="img:/icons/logo_app.svg" 

Any tips are really welcome, since I would like to submit the app to the store. :-)

Quasar: 2.1.1 
Cordova: 10.0.0

**Comment by ontwikkelfabriek** [comment]
@i-mage 

I would debug the app in either chrome inspector( will not work if build in prod mode I think) or you can always use android studio / xcode to debug/look at the logger( it will show errors for your app for sure) and figure out what is wrong.

Some other thoughts:

> icon="img:/icons/logo_app.svg"

Do import the svg explicitly in your vue file?
https://quasar.dev/vue-components/icon#svg-usage

You could try and see if svg inline does work:
https://quasar.dev/vue-components/icon#inlined-svg

**Comment by rstoenescu** [comment, marked as answer]
Try not using paths that begin with the forward slash.
`icon="img:/icons/logo_app.svg"` should be `icon="img:icons/logo_app.svg"`

**Reply by rstoenescu** [reply]
Quasar injects a `<base>` tag when it's needed (based on the quasar.conf) which provides a seamless experience for the /public folder, as long as you don't use a path that starts with "/" which forces the browser to discard that. https://next.quasar.dev/quasar-cli/handling-assets

**Reply by i-mage** [reply]
This solved the issue! Thanks!
